.github/workflows/yehezgun.yml

```diff
@@ -0,0 +1,38 @@
+name: Template Git Actions
```

Comment:
Why do we call this `Template Git Actions`?

Reply:
Because this repo is Next.jd 13 appDir starter template, so that's why I named it `Template Git Actions` which mean it's just a base template that can be changed/updated along the way the project grow (if using this template)